source/config/configSpec.py

```diff
@@ -54,6 +54,8 @@
 	WASAPI = featureFlag(optionsEnum="BoolFlag", behaviorOfDefault="enabled")
 	soundVolumeFollowsVoice = boolean(default=false)
 	soundVolume = integer(default=100, min=0, max=100)
+	silenceTimeSeconds = integer(default=30,min=0,max=3600)

```

Comment:
If you accept to replace "play silence" by "Keep audio device awake" in the User Guide and the GUI, you may also change it here for consistency:

```suggestion
	keepAudioAwakeTimeSeconds = integer(default=30,min=0,max=3600)
```